pkg/kubectl/resource/builder_test.go

```diff
@@ -608,6 +609,26 @@ func TestMultipleResourceByTheSameName(t *testing.T) {
 	}
 }
 
+func TestRequestModifier(t *testing.T) {
+	var got *rest.Request
+	b := NewBuilder(restmapper, LegacyCategoryExpander, scheme.Scheme, fakeClientWith("test", t, nil), corev1Codec).
```

Comment:
@smarterclayton you will need to fix `LegacyCategoryExpander` since you moved it in #55647